pkgs/development/tools/continuous-integration/buildkite-agent/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, makeWrapper, coreutils, git, openssh, bash, gnused, gnugrep }:
+
+stdenv.mkDerivation rec {
+  name = "buildkite-agent-2.1.8";
+
+  src = fetchurl {
+    url = "https://github.com/buildkite/agent/releases/download/v2.1.8/buildkite-agent-linux-386-2.1.8.tar.gz";
+    sha256 = "f54ca7da4379180700f5038779a7cbb1cef31d49f4a06c42702d68c34387c242";
+  };
+
+  buildInputs = [ makeWrapper ];
+  phases = "unpackPhase installPhase";
+  sourceRoot = ".";
+  installPhase = ''
+    install -Dt "$out/bin/" buildkite-agent
+
+    mkdir -p $out/share
+    mv hooks bootstrap.sh $out/share/
+
+    patchShebangs $out/share/
```

Comment:
Patching shebangs is handled by the fixup phase, which you've disabled. If you follow my suggestion above, please remove this and the patch below.